src/Raven.Client/Documents/Session/IAdvancedSessionOperations.Stream.cs

```diff
@@ -95,6 +106,50 @@ public partial interface IAdvancedSessionOperations
         ///     Returns the results of a query directly into stream 
         /// </summary>
         void StreamInto<T>(IRawDocumentQuery<T> query, Stream output);
+    }
+
+    public interface ITimeSeriesSessionStreamOperations<T>
+    {
+        IEnumerator<TimeSeriesStreamResult<T>> Stream(IQueryable<T> query);
+
+        IEnumerator<TimeSeriesStreamResult<T>> Stream(IQueryable<T> query, out StreamQueryStatistics streamQueryStats);
```

Comment:
stats?

Reply:
the name match to what we have in the normal stream.